src/stores/widgets/WidgetLayoutStore.ts

```diff
@@ -40,6 +40,7 @@ export enum Container {
 
     // ... more as needed. Note that most of this code assumes that there
     // are only two containers, and that only the top container is special.
+    Center = "center"
```

Comment:
@turt2live What do you think about the naming "center" here? I guess I am overall a bit surprised that we're using "directional" names in this whole enum instead of "functional" names like "primary content" (for the new one), "floating above content" (for top), etc. Left / right naming starts to feel especially strange once you think about i18n, which might place things on opposites sides for certain UI languages. If it were just an internal enum, then we could probably do whatever, but it feels a bit more awkward in this case since the values are emitted into room state when you share the layout.

As a meta-point, if "center" is okay, which spelling do we use...? 🤓

Reply:
I'd say for this is okay for now, as it seems like a proper continuation of the current scheme. I am still not convinced the overall scheme is the best one to use, but we can consider that separately from this work.